modules/heat_transfer/include/utils/FormulationEnums.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+/**
+ *  FormulationEnums contains the enumeration used to determine between
+ *  time or frequency formulations involving electromagnetic electical fields
+ *  and used to determine between solving for an electrostatic or
+ *  electromagnetic electric field
+ */
+namespace FM
```

Comment:
is this unique enough to be sufficiently future proof?

Reply:
No, that's a good point. I have changed it to `ElectromagneticFormulation`.